build/gulpfile.extensions.js

```diff
@@ -108,7 +121,7 @@ const tasks = compilations.map(function (tsconfigFile) {
 	const cleanTask = task.define(`clean-extension-${name}`, util.rimraf(out));
 
 	const compileTask = task.define(`compile-extension:${name}`, task.series(cleanTask, () => {
-		const pipeline = createPipeline(false, true);
+		const pipeline = createPipeline(sqlNonBuiltInExtensions.includes(name), true); // {{SQL CARBON EDIT}}
```

Comment:
there's a line in this function for ".pipe(filter(['**', '!**/*.nls.json']))" maybe that's why the commands are not localized now,. they need the nls.json

Reply:
as chatted offline.. I did try that. but that doesn't affect top level package.nls.json only the nls.json in out. I will try it again though, I might be missing something.